assets/hmbkp.js

```diff
@@ -139,21 +139,6 @@ var BackUpWordPressAdmin = (function($){
 			);
 		} );
 
-		jQuery( document ).one( 'click', '.hmbkp_send_error_via_email', function ( e ) {
```

Comment:
This doesn't seem like a CS/CI fix, should probably be double-checked to see if the removed code is not needed / does not need a replacement.